sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -140,6 +140,17 @@ def test_sidecar_workflow(self):
     from .testdata.sidecar import sidecar_pipeline
     self._test_pipeline_workflow(sidecar_pipeline, 'sidecar.yaml')
 
+  def test_loop_parallelism_workflow(self):
+    """
+    Test compiling a loop with parallelism defined workflow.
+    """
+    from .testdata.loop_static_with_parallelism import pipeline
+    self._test_pipeline_workflow(
+      pipeline,
+      'loop_static_with_parallelism.yaml',
+      normalize_compiler_output_function=lambda f: re.sub(
+          "(loop-item-param-[a-z 0-9]*|for-loop-for-loop-[a-z 0-9]*)", "with-item-name", f))
```

Comment:
`with-item-name` is not a valid parameter for this pipeline, is there a reason you need to substitute over here?

Reply:
Oops, I just copied other test cases and forgot to delete this, thanks.